actors/builtin/miner/miner_test.go

```diff
@@ -808,7 +850,35 @@ func (h *actorHarness) extendSector(rt *mock.Runtime, sector *miner.SectorOnChai
 		exitcode.Ok,
 	)
 	rt.Call(h.a.ExtendSectorExpiration, params)
+}
+
+func (h *actorHarness) reportConsensusFault(rt *mock.Runtime, from addr.Address, params *miner.ReportConsensusFaultParams, dealIDs []abi.DealID) {
+	rt.Reset()
```

Comment:
This mock reset is worrying. I expect reset to be used only in the test code itself, rather than here. On the other hand, there should be a `rt.Verify()` call after the last `rt.Call` in this method (line 881), which will have the effect of resetting after verifying expectations.

I looked into other uses of `Reset` that might have guided you and saw that I missed this in your tests earlier for extending sector expirations. I will submit a PR to fix up the patterns there.